File: day5/C++/Pattern_2.cpp
/**
  * @author : ashwek
  * @date : 26/12/2018
  */

#include<iostream>

using namespace std;

int main(){

    int n, k=1;

    cout<<"Enter n = ";
    cin>>n;

    for(int i=1; i <= n; i++){
        for(int j=1; j<=i; j++)
            cout<<k++ <<" ";
        cout<<endl;
    }
    return 0;
}

File: day5/Java/Pattern__2.java
/* @date 14/10/2020
 * @author Shashwat Gupta (shashwatxdevelop)
 */
import java.util.Scanner;
public class Pattern__2 {

	public static void main(String[] args) {
		int i,j,n, s=0;
		Scanner sc = new Scanner (System.in);
		n= sc.nextInt();
		for (i=1; i<=n; i++)
		{
			for (j=1; j<=i; j++)
			{
				s=s+1;
				System.out.print(s+" ");
			}
			System.out.println();
		}

	}

}
